.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,115 @@
+---
+name: Bug report
```

Nit to improve UX. Easy to spot in the list.
```suggestion
name: 🐛 Bug report
```